server/models/handlers/availability.js

```diff
@@ -0,0 +1,29 @@
+// Gets the time availabilities for a specific worker for a specific school.
+async function getWorkerAvailability(workerId, schoolId) {
+    const sampleAvailabileTimes = [
+        {
+            workerTimeslotId: 1,
+            startTime: '08:00:00', 
+            endTime: '08:30:00',
+            date: '2020-10-20'
+        },
+        {
+            workerTimeslotId: 2,
+            startTime: '08:30:00', 
+            endTime: '09:00:00',
+            date: '2020-10-20'
+        },
+        {
+            workerTimeslotId: 3,
+            startTime: '09:00:00', 
+            endTime: '09:30:00',
+            date: '2020-10-20'
+        }
+    ];
+
+    return sampleAvailabileTimes;
+};
+
+module.exports = {
```

Comment:
The current output is in the format: 
            workerTimeslotId: <value>
            startTime: <value>
            endTime: <value>
            date: <value>

Would this output only be for the current week? If a worker uploads their availability for the several upcoming months, would the output display all the worker's scheduled times? This could result in an extensive output if the worker has scheduled hundreds of slots in advance.
A potential workaround could be to create a condition where the output is only displayed between a certain time range (eg. only the slots available in the upcoming week) or only display the next 10 earliest slots available.

Reply:
We could even possibly pass in the date as a parameter to return only the availabilities for a specific date

Reply:
@nancyprabhakar for now I think we may be able to leave it as it is, but we may need to make changes in the next sprint, we can just simply return all the available time slots for a specific worker and their school

Reply:
Great points Nancy and Bradley! I have updated the endpoint to accept startTime and endTime as optional parameters to allow for more flexibility and greater variety of uses.